services/app-api/src/postgres/contractAndRates/prismaSharedContractRateHelpers.ts

```diff
@@ -248,7 +250,7 @@ function ratesRevisionsToDomainModel(
     }
 
     domainRevisions.sort(
-        (a, b) => a.createdAt.getTime() - b.createdAt.getTime()
+        (a, b) => a.rate.createdAt.getTime() - b.rate.createdAt.getTime()
```

Comment:
This preserves the rate order by createdAt in each revision. I noticed in the UI the rates were being sorted by the latest updated rate.